_source/handbook/02_drive.md

```diff
@@ -253,6 +253,20 @@ To accomplish this, just annotate those asset elements with `data-turbo-track="r
 </head>
 ```
 
+## Removing Assets When They Change
+
+As we saw above, Turbo Drive merges the contents of the `<head>` elements. When a page depends on CSS or JavaScript that other pages do not, it can be useful to remove them when navigating away from the page.
```

Comment:
Another use case worth mentioning is not triggering a full page reload when deploying changes that only affect styles.